code/game/machinery/portable_turret.dm

```diff
@@ -902,7 +902,7 @@ Status: []<BR>"},
 
 /obj/machinery/porta_turret_cover/attack_hand(mob/user)
 	. = ..()
-	if(.)
+	if(. || !Parent_Turret)

```

Comment:
This solution will likely deal with multiple edge cases simultaneously. Even if I did reliably reproduce the runtime (with no guarantee it is the issue on live) I would still implement this fix to deal with edge cases I'm not able to anticipate.